examples/sequence_tagging/train.py

```diff
@@ -156,7 +158,7 @@ def __init__(self, num_labels, name=None, *args, **kwargs):
             int(math.ceil((num_labels - 1) / 2.0)), "IOB")
         self.reset()
 
-    def add_metric_op(self, *args): 
+    def add_metric_op(self, *args):
```

Comment:
Can we use `fluid.layers.chunk_eval` here, is there any specific purpose to wrap a `Chunk_eval` class